source/general/index.rst

```diff
@@ -1,80 +1,87 @@
-=======
+.. _general_home:
+
+#########
 General
-=======
+#########
 
-How to download
-===============
+Welcome to the ProjectKorra Wiki, here you can find information regarding everything ProjectKorra!
 
-Contributing
+Background
 ============
+ProjectKorra is an organization, established by MistPhizzle in June 2014, to continue maintaining a version of Orion’s original Minecraft: The Last Airbender plugin for Bukkit. ProjectKorra’s purpose is to bring the Avatar: The Last Airbender and Avatar: The Legend of Korra universes into Minecraft, giving player an opportunity to experience the show in a new light.
 
-Setting up the environment
---------------------------
-
-To begin editing the wiki we first need to set our envrionment up. The prequirements needed is just `python <https://www.python.org/downloads/>`_.
-
-.. note:: When installing python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
-
-First thing you need to do is to fork the git repository and then clone it. 
-
-.. note:: Remember to replace *exampleusername* with your actual github username
-
-.. code:: bash
 
-    git clone https://github.com/exampleusername/ProjectKorraWiki.git
+Downloads
+===========
+The following are all of ProjectKorra's available resources. Only things listed here are official ProjectKorra resources, but there are also community-made addons you can get on our `forums`_!*
 
-The wiki is made with sphinx using the *sphinx_rtd_theme* from readthedocs. We've bundled all you need into one file.
-Navigate to the directory you cloned the project to, then we will install all the requirements by running the following command in terminal / command prompt
+* Vanilla
 
-.. code:: bash
+  * `Core`_
 
-    pip install -r .\requirements.txt
+* Modded
 
-This should install both sphinx as well as the theme.
+  * `ATLA Mod`_
 
-To check that sphinx is correctly install run:
+**\*PROJECTKORRA IS NOT RESPONSIBLE FOR ERRORS CAUSED BY NON PROJECTKORRA RESOURCES RESOURCES DOWNLOADED FROM OUTSIDE OF THE PROJECTKORRA FORUMS ARE ALSO NOT INSPECTED BY PROJECTKORRA AND MAY CONTAIN MALICIOUS CODE** 
```

Comment:
a. reread this
b. it'd look better as a note

Reply:
I don't like how the notes look, but fine.